tests/test_cli/test_cfg_subcmd.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+import shutil
+
+from test_cli import capture
+this_folder = os.path.dirname(__file__)
+
+def test_cfg_help(python_version):
+    """Test if Manim successfully adds configparsers when a subcommand is invoked."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg --help"
```

Comment:
**Praise** : This is really a nice workaround. this is way better than changing the CWD :D

Reply:
Thank you! I wouldn't have tried that if you didn't suggest it, so the credit is all yours :P